guide/src/fundamentals/relations.md

```diff
@@ -1,26 +1,29 @@
 # Relations
 
-A relation is a component which *links* to another `Entity`, similar to a foreign key in a database.
+A relation is a component which *links* to another `Entity`, similar to a foreign key in a database. This can be used to construct different kinds of graphs and trees inside the ECS.
 
-The linked entity is referred to as the `object` of a relation, while the entity the component is attached to is called the `subject`.
+The links between entities are managed by the ECS itself and will always be valid, see [Lifetime](#lifetime).
 
-This allows forming hierarchies such as *parent-child* relations for transforms and UI, as well as arbitrary graphs.
+The linked entity is referred to as the `target` of a relation, while the entity the component is attached to is called the `subject`.
 
-A relation is used as a *parameterized* component, which requires an `Entity` to be fully instantiated.
+This allows forming hierarchies such as *parent-child* relations for transforms and UI, as well as arbitrary graphs.
 
```

Comment:
Maybe here would be a good place to reinforce the target-subject terms after *parent-child*? Not sure.
Example:

`This allows forming hierarchies such as *parent-child* (target-subject) relations for transforms and UI...`

Not really sure because the relation can have any arbitrary direction

Reply:
Thank you, I've clarified it a bit more :)